parsec/utils.py

```diff
@@ -61,22 +61,22 @@
 # 80% in order to make sure that a clock shift is caught during the
 # handshake instead of being caught by another API call later on.
 
-BALLPARK_CLIENT_EARLY_OFFSET = 50.0  # seconds
-BALLPARK_CLIENT_LATE_OFFSET = 70.0  # seconds
+BALLPARK_CLIENT_EARLY_OFFSET = 50  # seconds
+BALLPARK_CLIENT_LATE_OFFSET = 70  # seconds
 BALLPARK_CLIENT_TOLERANCE = 0.8  # 80%
 
 
 def timestamps_in_the_ballpark(
     client: DateTime,
     backend: DateTime,
-    ballpark_client_early_offset: float = BALLPARK_CLIENT_EARLY_OFFSET,
-    ballpark_client_late_offset: float = BALLPARK_CLIENT_LATE_OFFSET,
+    ballpark_client_early_offset=BALLPARK_CLIENT_EARLY_OFFSET,
+    ballpark_client_late_offset=BALLPARK_CLIENT_LATE_OFFSET,
```

Comment:
Why not keep those arguments as floats?

Reply:
No need to be float, and it is compared to int (seconds) in the most of case